geodepy/tests/test_transform.py

```diff
@@ -3,10 +3,11 @@
 from geodepy.transform import (conform7,
                                conform14,
                                atrf2014_to_gda2020,
-                               gda2020_to_atrf2014,
+                               conform_gda2020_to_atrf2014,
+                               conform_atrf2014_to_gda2020,
                                mga94_to_mga2020,
                                mga2020_to_mga94)
-from geodepy.constants import itrf14togda20, gda94_to_gda2020
+from geodepy.constants import itrf2014_to_gda2020, gda1994_to_gda2020
```

Comment:
I think it best use stick with common usage. So, GDA94 but GDA2020. Similarly, the ITRF versions pre-2000 have only two digit years (ITRF92), while those from 2000 on have four digits (ITRF2014).